spec/models/video_spec.rb

```diff
@@ -51,4 +54,30 @@
       expect(SendVideoStatusWebhookJob).to have_been_enqueued.exactly(0)
     end
   end
+
+  describe 'get total stream time' do
+    it 'returns a the sum of all duration for a video\'s stream time' do
```

Comment:
```suggestion
    it "returns the sum of all duration for a video's stream time" do
```

It's okay to use double quotes when you need to escape a `'` (I learned that from Larry, that told me my escapes were hurting his eyes 😅 ).